web/annotation_editor_layer_builder.css

```diff
@@ -123,6 +124,10 @@
   cursor: auto;
 }
 
+.annotationEditorLayer .inkEditorCanvas {
+  touch-action: none;
```

Comment:
*I've not had time to really review this yet, so just one quick question for now:*

Could this rule not be added to the `.annotationEditorLayer .inkEditor .inkEditorCanvas`-block further down in this file, since that'd (ever so slightly) reduce the size increase of this file?